config/game_options.txt

```diff
@@ -394,6 +394,8 @@ JOIN_WITH_MUTANT_RACE
 
 ## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
 ROUNDSTART_RACES human
+ROUNDSTART_RACES space_elf
+ROUNDSTART_RACES klingoff
```

Comment:
Not humans. But let me guess this is here because they're the same as humans?

Reply:
mhm